### ISSUE TITLE
Implement training loop with ConVSE loss

### ISSUE THREAD

**Issue by iremurhan** [iremurhan/pytorch_simclr#1]
Deadline: 30.04.2025
